A51-custom-backend-metrics.md

```diff
@@ -349,6 +368,12 @@ public class MetricRecorder {
 
   // Clear the queries-per-second metrics data.
   public void clearQpsMetric();
+
+  // Update the errors-per-second metrics data.
+  public void putEpsMetric(double value);
```

Comment:
This should be `setEpsMetric(double)`. We use set single entries and put for mapped entries. That's why only `putUtilizationMetric(key, value)` uses put in its name.

Reply:
Forgot to update this here as well, fixed.